cardano-api/src/Cardano/Api/Address.hs

```diff
@@ -466,6 +466,15 @@ data StakeCredential
        | StakeCredentialByScript  ScriptHash
   deriving (Eq, Ord, Show)
 
+instance ToJSON StakeCredential where
+  toJSON =
+    Aeson.object .
+    \case
+      StakeCredentialByKey keyHash ->
```

Comment:
So we use camel case for `ToJSON` instances defined in cardano-api. We should write the keys as `stakingKeyHash` and `stakingScriptHash`. You're also missing an indent in front of `\case`.

Reply:
done